exporter/context.go

```diff
@@ -143,13 +152,52 @@ func newImportContext(c *common.DatabricksClient) *importContext {
 		allDirectories:      []workspace.ObjectStatus{},
 		allWorkspaceObjects: []workspace.ObjectStatus{},
 		workspaceConfKeys:   workspaceConfKeys,
+		shImports:           make(map[string]bool),
 	}
 }
 
+func getLastRunString(fileName string) string {
```

Comment:
can you also parse durations, e.g. `1w`, `1M`, `1d`, `1h` - the equivalents of `time.Duration.String`.

Reply:
Maybe, let see if it will be required - I want to get feedback from customer on UX. But really, the duration here doesn't matter much, maybe except the case when it's scheduled from cron, but then it should be provided via `updated-since` parameter, not from file